tests/expected/configured/save_config.yaml

```diff
@@ -2,6 +2,12 @@ cache_size_mb: 8
 keep_alive: 75
 listen_addresses: localhost:3111
 worker_processes: 1
+cors:
```

Comment:
do we want to actually output `cors` here if there are no configurations?  Esp when we have `max_age: null` style params